src/hydra-queue-runner/queue-monitor.cc

```diff
@@ -569,7 +573,7 @@ Step::ptr State::createStep(ref<Store> destStore,
 
 
 Jobset::ptr State::createJobset(pqxx::work & txn,
-    const std::string & projectName, const std::string & jobsetName)
+    const std::string & projectName, const std::string & jobsetName, JobsetID jobsetID)
```

Comment:
```suggestion
    const std::string & projectName, const std::string & jobsetName, const JobsetID jobsetID)
```
to match the declaration, everything else looks fine C++-wise :)

Reply:
Thanks, I fixed this and squashed them all in to one commit. I appreciate your review!